userdocs/src/index.md

```diff
@@ -1,33 +1,22 @@
----
-title: Introduction
-type: docs
----
+# `eksctl` - The official CLI for Amazon EKS
 
-<div id="github"><script async defer src="https://buttons.github.io/buttons.js"></script><a class="github-button" href="https://github.com/weaveworks/eksctl" data-icon="octicon-star" data-show-count="true" aria-label="Star weaveworks/eksctl on GitHub">Star</a><div id="github-section"><a class="github-button" href="https://github.com/weaveworks/eksctl/fork" data-icon="octicon-repo-forked" data-show-count="true" aria-label="Fork weaveworks/eksctl on GitHub">Fork</a></div></div>
+sponsored by [![Weaveworks](img/weaveworks.svg)](https://www.weave.works/) and built by [![Contributors](img/gophers.png)](https://github.com/weaveworks/eksctl/graphs/contributors) on [![Github](img/octocat.svg)](https://github.com/weaveworks/eksctl)
 
-# **`eksctl` - The official CLI for Amazon EKS**
+`eksctl` is a simple CLI tool for creating clusters on EKS - Amazon's new managed Kubernetes service for EC2.
+It is written in Go, uses CloudFormation, was created by [Weaveworks](https://www.weave.works/) and it welcomes
+contributions from the community. Create a basic cluster in minutes with just one command:
 
-### sponsored by [![Weaveworks](introduction/images/weaveworks.svg#inline-ww)](https://www.weave.works/) and built by [![Contributors](introduction/images/gophers.png#inline)](https://github.com/weaveworks/eksctl/graphs/contributors) on [![Github](introduction/images/octocat.svg#inline)](https://github.com/weaveworks/eksctl)
-
-`eksctl` is a simple CLI tool for creating clusters on EKS - Amazon's new managed Kubernetes service for EC2. It is
-written in Go, uses CloudFormation, was created by [Weaveworks](https://www.weave.works/) and it welcomes
-contributions from the community. Create a basic cluster in minutes with just one command :
-
- <h2 id="intro-code">**`eksctl create cluster`**</h2>
-
-A cluster will be created with default parameters
-
-> ∙ exciting auto-generated name, e.g. "fabulous-mushroom-1527688624"
-
-> ∙ 2x `m5.large` nodes (this instance type suits most common use-cases, and is good value for money)
-
-> ∙ use official AWS EKS AMI
-
-> ∙ `us-west-2` region
-
-> ∙ dedicated VPC (check your quotas)
+```console
+eksctl create cluster
```

Comment:
Dont we want this to be `<h2 id="intro-code">**`eksctl create cluster`**</h2>` ?

Reply:
That'd mean bringing back some of the CSS we had before - I'll re-add but will need some help figuring this out? cc @bia

Reply:
I see. I don't think it's super important. I'd rather have this merged ASAP.

Reply:
I'm fixing this now

Reply:
Just going through all of [my old changes](https://github.com/weaveworks/eksctl/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+author%3Abia) and re-adding them.

Reply:
@bia and I just decided to move this work to a subsequent PR. Just tying things up for merging this.